embedded/embedded-api/src/main/java/org/mule/runtime/module/embedded/internal/Serializer.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.runtime.module.embedded.internal;
+
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+import java.io.OutputStream;
+import java.io.Serializable;
+
+public class Serializer {
```

Comment:
yes, but since this is meant to be used in other java envs, we are trying to keep at a minimum the number of external dependencies this project dependes on.